server/services/reportedAdjudicationsService.ts

```diff
@@ -519,4 +587,26 @@ export default class ReportedAdjudicationsService {
       prisonerFullName: formatName(prisoner.firstName, prisoner.lastName),
     }
   }
+
+  async filterAdjudicationsByLocation(
+    adjudications: ReportedAdjudicationEnhancedWithIssuingDetails[],
+    chosenLocationId: LocationId,
+    user: User
+  ) {
+    const location = await (
+      await this.locationService.getLocationsForUser(user)
+    ).filter(loc => loc.locationId === chosenLocationId)
+    const { locationPrefix } = location[0]
+    return adjudications.filter(adj => this.getLocationPrefix(adj.prisonerLocation) === locationPrefix)
+  }
+
+  getLocationPrefix = (locationCode: string) => {
+    const parts = locationCode && locationCode.split('-')
+    if (parts && parts.length > 0) {
+      if (parts[1] === 'RECP') return `${parts[0]}-RECEP`
```

Comment:
This bit is interesting - needs to be tested in preprod... the prefix for reception in Dev MDI is different in the userDescription from the data actually labelled as prefix. Unusual - hoping it's just dodgy dev data!